core/src/main/resources/xtdb/sql/parser/sql.ebnf

```diff
@@ -380,6 +380,10 @@ schema_name
     : [ schema_name <period> ] !#'(?i)(\b(SYSTEM_TIME|VALID_TIME)\b)' identifier
     ;
 
+(* Following the lead of regular_identifier here, filtering known function calls *)
+routine_identifier
```

Comment:
Essentially looking at EVERY other potential `fn_name()` type invocation in the grammar and explicitly checking that the routine name is NOT one of those.